js/app-super-resolution/README.md

```diff
@@ -0,0 +1,59 @@
+# RUNNING SUPER-RESOLUTION ON IMAGES ACROSS VARIOUS PLATFORMS USING ONNX RUNTIME MODELS
+## Examples
+1. MOBILE
+  <img width="318" alt="image" src="https://user-images.githubusercontent.com/106185642/181639530-9c808167-d68c-49d4-8e89-72aeeb11164e.png">
+
+2. WEB
+  <img width="1215" alt="image" src="https://user-images.githubusercontent.com/106185642/181638855-f341e52e-dfc1-4362-b93a-0117f0cfd65a.png">
+   
+
+## Steps of building project from scratch
+0. Prerequisites
+    1. Install Node.js
+    2. Install Java and OpenJDK
+    3. Install expo
```

Comment:
node.js/java, etc. maybe common for people,

but for expo fairly new concept, can add link to the name, for people to conveniently click and check. also doesn't hurt to provide links for openjdk, etc. ( for example, is there a specific version required to install?)

Reply:
sounds good, adding that